Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -479,17 +479,20 @@ def generate_impl(serialized_types, serialized_enums, headers):
                     result.append('#endif')
             result.append('    return (value.toRaw() | allValidBitsValue) == allValidBitsValue;')
         else:
-            result.append('    switch (static_cast<' + enum.namespace_and_name() + '>(value)) {')
-            for valid_value in enum.valid_values:
-                if valid_value.condition is not None:
-                    result.append('#if ' + valid_value.condition)
-                result.append('    case ' + enum.namespace_and_name() + '::' + valid_value.name + ':')
-                if valid_value.condition is not None:
-                    result.append('#endif')
-            result.append('        return true;')
-            result.append('    default:')
-            result.append('        return false;')
-            result.append('    }')
+            if len(enum.valid_values) == 0:
```

Comment:
This shouldn't be necessary.  We shouldn't be serializing enums with 0 valid values.